.github/workflows/workflow.yml

```diff
@@ -2,7 +2,7 @@ name: Checks and Tests
 'on':
   push:
     branches:
-      - develop
+      - 'release/**'
```

Comment:
This is incorrect.

These actions are to be done in all PRs going into `master`.